src/gtp/mod.rs

```diff
@@ -143,14 +166,20 @@ impl<'a> GTPInterpreter<'a> {
             },
             "genmove"          => {
                 let color = Color::from_gtp(command[1]);
-                let m  = self.engine.gen_move(color, &self.game);
+                self.timer.start();
+                let budget = self.timer.budget(&self.game);
+                let mut stream = stderr();
+                stream.write_line(format!("Thinking for {}ms", budget).as_slice());
+                stream.write_line(format!("{}ms time left", self.timer.main_time_left()).as_slice());
```

Comment:
OK, so I'll merge it like this. It would probably be worth to encapsulate this into a `stderr!` macro at some point as we will probably use this more often (e.g. for [GoGui graphics output](http://gogui.sourceforge.net/doc/live-gfx.html).